lib/generators/boring/ci/github_action/install/install_generator.rb

```diff
@@ -5,7 +5,7 @@ module Ci
     module GithubAction
       class InstallGenerator < Rails::Generators::Base
         desc "Adds Github Action to the application"
-        source_root File.expand_path("templates", __dir__)
+        e_root File.expand_path("templates", __dir__)sourc
```

Comment:
what is the reason for changing this

Reply:
It was changed automatically in the repo. I did not do this.